content/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -657,13 +516,13 @@ export const getServerSideProps: GetServerSideProps<
         getImages({
           params: queryParams(sectionName, conceptResponse),
           toggles: serverData.toggles,
-          pageSize: 5,
+          pageSize: 12,
```

Comment:
This affects current pages as well, if that wasn't intended, you can get access to the toggle values through `serverData`!